src/main/java/hudson/plugins/parameterizedtrigger/AbstractBuildParameterFactory.java

```diff
@@ -24,10 +22,10 @@
      *      The build which the parameterized trigger is configured and executing.
      * @param listener
      *      Connected to the build output.
-     *      
+     *
      * @return can be empty but never null.
      */
-    public abstract List<AbstractBuildParameters> getParameters(AbstractBuild<?,?> build, TaskListener listener)
+    public abstract List<AbstractBuildParameters> getParameters(AbstractBuild<?, ?> build, TaskListener listener)
```

Comment:
Please avoid such formatting changes if you don't change the line. They make the review and merges more difficult